### ISSUE TITLE
Feature Request: Buff Unit Search

### ISSUE THREAD

**Issue by Darwe-Canine** [lyrgard/ffbeEquipNG#7]
Request to add search/filter for Buffs to the Unit Search page.

**Comment by lyrgard** [comment]
Yes, improving the unit search with new feature is something that is itching me a lot... It will have to wait for v2, but I'll definitely work on it !

I want to make the ultimate unit search tool !
